mediapipe/tasks/python/components/containers/category.py

```diff
@@ -39,10 +39,10 @@ class Category:
     category_name: The label of this category object.
   """
 
-  index: int
-  score: float
-  display_name: str
-  category_name: str
+  index: Optional[int] = None
```

Comment:
As per category.proto these fields were supposed to be [optional](https://github.com/google/mediapipe/blob/master/mediapipe/tasks/cc/components/containers/proto/category.proto). Hence including these changes.